src/_index.js

```diff
@@ -356,13 +356,35 @@ class EasySelect{
         if(this.isDisabled) return false;
 
         // avoid duplicate value
-        if(this.selectTagData.filter(option => option.value === value).length > 0){
-            if(this.config.warning) console.warn(`[ES] ${value} will not be added due to duplicating`);
-            return false;
+        const isDuplicate = (value) => {
```

Comment:
After your changes, does this `add()` function exit when a duplicate value is found?